langchain_postgres/vectorstores.py

```diff
@@ -702,13 +703,24 @@ def _handle_field_filter(
             if operator in {"$in"}:
                 return queried_field.in_([str(val) for val in filter_value])
             elif operator in {"$nin"}:
-                return queried_field.nin_([str(val) for val in filter_value])
+                return ~queried_field.in_([str(val) for val in filter_value])
             elif operator in {"$like"}:
                 return queried_field.like(filter_value)
             elif operator in {"$ilike"}:
                 return queried_field.ilike(filter_value)
             else:
                 raise NotImplementedError()
+        elif operator == "$null":
```

Comment:
@eyurtsev let me know if you think that this operator should be called something like "$exists"

Reply:
Yes, would you mind updating to $exists.